source/apps/crawl_space/models.py

```diff
@@ -69,23 +70,15 @@ def get_absolute_url(self):
         return reverse('base:project',
             kwargs=dict(project_slug=self.project.slug))
 
-
     def save(self, *args, **kwargs):
-
         if self.pk is None:
-            super(CrawlModel, self).save(*args, **kwargs)
-
+            self.slug = slugify(self.name)
+            # TODO:
+            # Another weird call with a side effect that has to be fixed.
             model_path = self.ensure_model_path()
```

Comment:
Then I don't think that's a weird side effect, that's what the function should be doing.